src/test/java/fi/aalto/cs/intellij/common/ResourcesTest.java

```diff
@@ -0,0 +1,71 @@
+package fi.aalto.cs.intellij.common;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import org.junit.Test;
+
+public class ResourcesTest {
+
+  @Test
+  public void testGetProperties() {
+    final MockInputStream stream = new MockInputStream("a=x\nb=y\n");
+
+    Resources res = new Resources(name -> {
+      assertEquals("correct-properties", name);
+      return stream;
+    });
+
+    assertFalse(stream.hasCloseMethodBeenCalled);
+
+    Properties props = res.getProperties("correct-properties");
+
+    assertTrue(stream.hasCloseMethodBeenCalled);
+
+    assertNotNull(props);
+    assertEquals("x", props.getProperty("a"));
+    assertEquals("y", props.getProperty("b"));
+  }
```

Comment:
I'm a bit skeptical about the [MockInputStream approach](https://github.com/Aalto-LeTech/intellij-plugin/pull/51/files#diff-15e23dbb3e8a5b277f36c9663056e8edR58-R69), but I can't find a way of using `Mockito.verify(...)` or `Mockito.spy(...)`, but I'm still looking into this